code/modules/client/preferences_savefile.dm

```diff
@@ -315,6 +315,7 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	S["feature_lizard_body_markings"]	>> features["body_markings"]
 	S["feature_lizard_legs"]			>> features["legs"]
 	S["feature_moth_wings"]				>> features["moth_wings"]
+	S["feature_moth_markings"]			>> features["moth_markings"]
```

Comment:
Can this be at the bottom of the savefile `S[` list, and be tagged as a Kepler change? Savefile conflicts are fucking scary because one messup can destroy everyones characters.